Assets/New Babylon/Scripts/InventoryTools/Seed.cs

```diff
@@ -44,10 +59,38 @@ public class Seed : MonoBehaviour {
 	}
 	
 	void Update () {
+        //counting days is hard work (for spawned seeds only)

```

Comment:
Maybe have a global day variable or something that maintains state to help track this.

Reply:
I would recomend looking at the observe observable pattern. Very useful here.